docs/docs/en/server/on-premise/install.md

```diff
@@ -125,12 +136,24 @@ Once the app is created you'll need to set the following environment variables:
 
 #### S3-compliant storages
 
-The environment variables required to authenticate against S3-compliant storages aligns with the [conventions set by AWS](https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-envvars.html) (e.g. `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_REGION`, `AWS_ENDPOINT`). Additionally, you need to set the `TUIST_S3_BUCKET_NAME` environment variable to indicate the bucket where the artifacts will be stored.
-
-> [!NOTE] RUST SDK
-> Tuist uses this [Rust SDK](https://github.com/durch/rust-s3), which you can use as a reference to understand how the environment variables are used.
+You can use any S3-compliant storage provider to store artifacts. The following environment variables are required to authenticate and configure the integration with the storage provider:
 
-### Google Cloud Storage
+| Environment variable | Description | Required | Default | Example |
+| --- | --- | --- | --- | --- |
+| `AWS_ACCESS_KEY_ID` | The access key ID to authenticate against the storage provider | Yes | | `AKIAIOSFOD` |
+| `AWS_SECRET_ACCESS_KEY` | The secret access key to authenticate against the storage provider | Yes | | `wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY` |
+| `AWS_REGION` | The region where the bucket is located | Yes | | `us-west-2` |
+| `AWS_ENDPOINT` | The endpoint of the storage provider | Yes | | `https://s3.us-west-2.amazonaws.com` |
```

Comment:
We also have these same variables with `TUIST_S3_*` prefix – these are generally preferred. Should we add that both variants are possible? In case, an on-premise user uses non-aws storage solution, using `AWS_*` for configuration would be a bit weird.